src/settings.js

```diff
@@ -55,17 +55,27 @@ const locales = {
   CS_CZ: 'cs_CZ', // Czech (Czech Republic)
   DA_DK: 'da_DK', // Danish (Denmark)
   DE_DE: 'de_DE', // German (Germany)
+  EN_GB: 'en_GB', // English (Great Britain)
```

Comment:
Just realized I change the comment here from "United Kingdom" to "Great Britain", if it matters